canonical-websites/www.ubuntu.com
# 2754: Build /openstack/install/build-openstack page

## Done

- Moved /download/cloud/build-openstack to /openstack/install/build-openstack
- Updated copy according to [copydoc](https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit#heading=h.pt6iw0mepowo)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/install/build-openstack
- Check that it matches the [copydoc](https://docs.google.com/document/d/1eQ16jzv6YRGzG_k_C-tUnhM06k6_Ic6fZM-x_ARMzuc/edit#heading=h.pt6iw0mepowo)
- Check that /download/cloud/build-openstack redirects to /openstack/install/build-openstack

## Issue / Card

Fixes #2754
